scripts/cdc_who_gh_comparison/run.py

```diff
@@ -0,0 +1,338 @@
+
+
+def format_cdc_data(data: list[dict[str, str|int|None]]) -> dict[str, int]:
+	logging.info("Formatting CDC data")
+	clean_data = {}
+	for entry in data:
+		if entry["Location"] in ["Non-US Resident", "Puerto Rico"]:
+			continue
+		clean_data[entry["Location"]] = entry["Cases"]
+	return clean_data
+
+
+def format_who_data(data: list[dict[str, str|int|None]]) -> dict[str, int]:
+	logging.info("Formatting WHO data")
+	clean_data = {}
+	for entry in data:
+		name = entry["COUNTRY"]
+		if "WHO" in name and "Region" in name:
+			continue
+		if name in ["USA", "UNITED KINGDOM"]:
+			continue
+		for who_name, gh_name in WHO_TO_GH.items():
+			if name.title() == who_name:
+				clean_data[gh_name] = entry["CasesAll"]
+		clean_data[name.title()] = entry["CasesAll"]
+	return clean_data
+
+
+def format_gh_usa_data(data: list[dict[str, str|int|None]]) -> dict[str, int]:
+	logging.info("Formatting G.h USA data")
+	gh_state_counts = {}
+	for entry in data:
+		if entry["Country"] == "United States":
+			for state in STATES:
+				if state in entry["Location"]:
+					if state in gh_state_counts:
+						gh_state_counts[state] += 1
+					else:
+						gh_state_counts[state] = 1
+					continue
+
+	return gh_state_counts
+
+
+def format_gh_global_data(data: list[dict[str, str|int|None]]) -> dict[str, int]:
+	logging.info("Formatting G.h global data")
+	gh_global_counts = {}
+	for entry in data:
+		confirmed = entry.get("confirmed")
+		if len(confirmed) == 0:
+			confirmed = 0
+		deaths = entry.get("death")
+		if len(deaths) == 0:
+			deaths = 0
+		gh_global_counts[entry["Country"]] = int(confirmed) + int(deaths)
+	return gh_global_counts
+
+
+def compare_cdc_data(gh_data: dict[str, int], cdc_data: dict[str, int]) -> dict[str, int]:
+	logging.info("Comparing CDC and G.h USA data")
+	diffs = {}
+	for state, cdc_count in cdc_data.items():
+		logging.info(f"{state} CDC count: {cdc_count} G.h count: {gh_data.get(state, 0)}")
+		diff = int(cdc_count) - gh_data.get(state, 0)
+		if diff:
+			diffs[state] = diff
+	if cdc_missing := gh_data.keys() - cdc_data.keys():
+		for state in cdc_missing:
+			diffs[state] = -gh_data.get(state, 0)
+	return diffs
+
+
+def compare_who_data(gh_data: dict[str, int], who_data: dict[str, int]) -> dict[str, int]:
+	logging.info("Comparing WHO and G.h global data")
+	diffs = {}
+	for country, who_count in who_data.items():
+		logging.info(f"{country} WHO count: {who_count} G.h count: {gh_data.get(country, 0)}")
+		diff = who_count - gh_data.get(country, 0)
+		if diff > 0:
+			diffs[country] = diff
+	return diffs
+
+
+def change_gh_data(changes: dict[str, int], dry_run: bool):
+	logging.info("Changing G.h USA data")
+	client = pygsheets.authorize(service_account_env_var="GOOGLE_CREDENTIALS")
+	client.set_batch_mode(True)
+	spreadsheet = client.open_by_key(DOCUMENT_ID)
+
+	sheet = spreadsheet.worksheet("title", LINE_LIST_SHEET)
+
+	# Finding column indices based on values breaks when using enums...
```

Comment:
This is likely to break -- is there a workaround to use column names?

Reply:
No, unfortunately this is the workaround for the `find` function breaking